src/android/app/src/main/res/values/strings.xml

```diff
@@ -481,13 +484,19 @@
     <string name="memory_petabyte">PB</string>
     <string name="memory_exabyte">EB</string>
 
-    <!-- Theme Modes -->
+    <!-- System theme -->
     <string name="change_theme_mode">Change Theme Mode</string>
     <string name="theme_mode_follow_system">Follow System</string>
     <string name="theme_mode_light">Light</string>
     <string name="theme_mode_dark">Dark</string>
+
+    <!-- Material You theme -->
     <string name="material_you">Material You</string>
-    <string name="material_you_description">Use system colors across the app</string>
+    <string name="material_you_description">Use system colors across the app (Will override (static) theme color option)</string>
+
+    <!-- Static theme color -->
+    <string name="static_theme_color">Change Theme Color</string>
```

Comment:
Consider if it may be worth changing the "Change Theme Mode" and "Change Theme Color" strings to simply "Theme Mode" and "Theme Color"